packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -47,6 +47,17 @@ export function errorHandling(errorDetails: any, label?: string, moreInfo?: stri
 
     if (errorDetails.mDetails !== undefined) {
         httpErrCode = errorDetails.mDetails.errorCode;
+        // open config file for missing hostname error
+        const msg = errorDetails.toString();
+        if (msg.includes("hostname")) {
+            if (ProfilesCache.getConfigInstance().usingTeamConfig) {
+                vscode.window.showErrorMessage("Required parameter 'hostname' must not be blank");
```

Comment:
I am trying this scenario with a config file without host specified but I don't get this error message. 

Instead, I see this:
<img src="https://user-images.githubusercontent.com/45845502/133614650-3a63b8f8-5093-4267-8a11-f08cf8a988d4.png" width="400" />

Reply:
It looks like it's coming from `const updSession = await ZoweExplorerApiRegister.getMvsApi(loadProfile).getSession();` from the promptCredentials function

Reply:
This hostname error seems to be appearing appropriately now. However, the property name in the config file is "host" (Zowe CLI version 7.0.0-next.202108202027), so it might be clearer if we change "hostname" to "host" in the message:

<img width="639" alt="Screen Shot 2021-09-28 at 9 34 48 PM" src="https://user-images.githubusercontent.com/45975633/135189071-72b0733c-3212-4869-aa78-b7faf2a724ec.png">